### DISCUSSION TITLE
i can use your api for free ?

### DISCUSSION THREAD

**Discussion by joukhar** [lrstanley/geoip#16] [Q&A] [answered]
i can use your api for free ?

and the rate limit is 15000 per day or month
and thanks your project is amazing

**Comment by lrstanley** [comment, marked as answer]
Hello!

You're more than welcome to utilize it. Do note that I am actively working on a rewrite (of both the frontend and API, including utilizing an OpenAPI spec, so clients can be generated in other languages), so some of the API endpoints will be adjusted soon. However, I will be maintaining 95% backwards compatibility with the existing API. Keep an eye out for this within the next couple of weeks.

As far as the rate limit, I believe I have it configured for 15,000/hour. This may be lowered in the future, but not lower than 2,500/hour, so as long as your calls would fit within that timeframe, you should be fine.

**Reply by joukhar** [reply]
You're amazing ,  15,000/hour is  more than enough , i want just to start a small website.
keep going  and thank you so much.
